Library/Homebrew/cask/audit.rb

```diff
@@ -663,16 +641,9 @@ def audit_github_repository_archived
 
       metadata = SharedAudits.github_repo_data(user, repo)
       return if metadata.nil?
-
       return unless metadata["archived"]
 
-      message = "GitHub repo is archived"
-
-      if cask.discontinued?
-        add_warning message
-      else
-        add_error message
-      end
+      add_error("GitHub repo is archived", strict_only: cask.discontinued?)
```

Comment:
Shouldn't this be `!cask.discontinued?`? Naming this `strict_only` somehow made it more confusing.

If the cask is discontinued, I don't think this should ever be an error.

Reply:
I preferred the old name too. 😬 

The `strict_only` naming is what used to be a warning. So if the cask is discontinued then there'll be a message show up if `brew audit --strict` is run.

I think this is the same behaviour as before (if you look at the lines before in the diff and translate the logic to the new names).

Reply:
But now we only have errors, no warnings, right? So I don't think this should be an error, even with `--strict`.

Reply:
Ohhhh. Right.